iamproperty/iamproperty.github.io
# Ds 69/checkbox

## Summary of Changes:
1. Modify the vue component
2. Update bootstrap variables to modify the CSS

## Rationale
Add checkbox as an option in the input component

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /elements/form/checkbox

## Ticket(s)
DS-69

## Checklist

## Notes
